cypress/e2e/acceptance/birth/birth-search.cy.js

```diff
@@ -87,6 +98,17 @@ describe('Birth search', () => {
       BirthSearchPage.noSurname();
     });
   });
+  describe('with a surname more than 30 character length', () => {
+    before(() => {
+      BirthSearchPage.visit();
+      BirthSearchPage.performSearch({
+        surname: 'surnamemorethanthirtycharacterlimit', forenames: 'forenamemorethanthirtycharacter', day: '01', month: '10', year: '2010'
```

Comment:
Similar comment to my earlier one, but could we instead use Pascal or camelCase for the `surname` attribute?